coalib/output/ConfigConverter.py

```diff
@@ -0,0 +1,162 @@
+import re
+
+from coalib.output.ConfWriter import ConfWriter
+from tomlkit import dumps
+from tomlkit import table, key, document
+from tomlkit.items import Key, KeyType
+from coalib.output.TomlConfWriter import TomlConfWriter
+
+
+class ConfigConverter:
+
+    def __init__(self, file):
+        self.__key_delimiter = ','
+        self.file = file
+        self.document = document()
+        self.unsavable_keys = ['save', 'toml_config']
+
+    def coafile_to_toml(self, sections):
+        """
+        Creates a toml document from coafile
+        sections
+
+        :param sections: sections obtained from coafile
+        """
+        comment_regex = re.compile(r'comment[\d]+')
+
+        if 'default' in sections.keys():
+            if not sections['default'].contents == {}:
```

Comment:
lets wrap this into an internal function, you are using this check for delete below too.